src/main/java/com/bulkbuy/enterprise/BulkBuyController.java

```diff
@@ -15,14 +19,35 @@ public String index() {
         return "start";
     }
 
+    /**
+     * Endpoint handler for /newOrder
+     * @return the page newOrder.html
+     */
     @RequestMapping("/newOrder")
-    public String newOrder(){
+    public String newOrder(Model model){
+        Order order = new Order();
+        model.addAttribute(order);
         return "newOrder";
     }
 
+    /**
+     * Endpoint handler for /orderLookup
+     * @return the page orderLookup.html
+     */
     @RequestMapping("/orderLookup")
-    public String orderLookup(){
+    public String orderLookup(Model model){
+        Order order = new Order();
+        model.addAttribute(order);
```

Comment:
I've added a model to the orderLookup endpoint to prepare the page for Thymeleaf usage when needed.